config/cesm/config_grids.xml

```diff
@@ -41,6 +41,9 @@
       <grid name="wav"	  compset="SWAV"  >null</grid>
       <grid name="wav"	  compset="DWAV"  >ww3a</grid>
       <grid name="wav"	  compset="WW3"	  >ww3a</grid>
+      <grid name="rof"    compset="XROF"  >r05</grid>
```

Comment:
Can you please reorder these so that the rof and glc grids are together with the other rof and glc grids.